mods/ctl/src/commands/providers/update.ts

```diff
@@ -81,18 +81,6 @@ export default class UpdateCommand extends Command {
         choices: ["tcp", "udp"],
         default: provider.transport
       },
-      {
```

Comment:
We must keep the prompts in the CTL and the backend will decide whether or not to ignore it.